CODEOWNERS

```
@@ -25,3 +25,337 @@ trainer.py @muellerzr @SunMarc
+/src/transformers/models/dac/*_modeling* @eustlb
+/src/transformers/models/encodec/*_modeling* @eustlb
+/src/transformers/models/hubert/*_modeling* @eustlb
+/src/transformers/models/mctct/*_modeling* @eustlb
+/src/transformers/models/mimi/*_modeling* @eustlb
+/src/transformers/models/mms/*_modeling* @eustlb
+/src/transformers/models/moshi/*_modeling* @eustlb
+/src/transformers/models/musicgen/*_modeling* @eustlb
+/src/transformers/models/musicgen_melody/*_modeling* @eustlb
+/src/transformers/models/pop2piano/*_modeling* @eustlb
+/src/transformers/models/seamless_m4t/*_modeling* @eustlb
+/src/transformers/models/seamless_m4t_v2/*_modeling* @eustlb
+/src/transformers/models/sew/*_modeling* @eustlb
+/src/transformers/models/sew-d/*_modeling* @eustlb
+/src/transformers/models/speech_to_text/*_modeling* @eustlb
+/src/transformers/models/speech_to_text_2/*_modeling* @eustlb
+/src/transformers/models/speecht5/*_modeling* @eustlb
+/src/transformers/models/unispeech/*_modeling* @eustlb
+/src/transformers/models/unispeech-sat/*_modeling* @eustlb
+/src/transformers/models/univnet/*_modeling* @eustlb
+/src/transformers/models/vits/*_modeling* @eustlb
+/src/transformers/models/wav2vec2/*_modeling* @eustlb
+/src/transformers/models/wav2vec2-bert/*_modeling* @eustlb
+/src/transformers/models/wav2vec2-conformer/*_modeling* @eustlb
+/src/transformers/models/wav2vec2_phoneme/*_modeling* @eustlb
+/src/transformers/models/wavlm/*_modeling* @eustlb
+/src/transformers/models/whisper/*_modeling* @eustlb
+/src/transformers/models/xls_r/*_modeling* @eustlb
+/src/transformers/models/xlsr_wav2vec2/*_modeling* @eustlb
+
+# Video models
+/src/transformers/models/timesformer/*_modeling* @Rocketknight1
+/src/transformers/models/videomae/*_modeling* @Rocketknight1
+/src/transformers/models/vivit/*_modeling* @Rocketknight1
+
+# Multimodal models
+/src/transformers/models/align/*_modeling* @zucchini-nlp
+/src/transformers/models/altclip/*_modeling* @zucchini-nlp
+/src/transformers/models/aria/*_modeling* @zucchini-nlp
+/src/transformers/models/blip/*_modeling* @zucchini-nlp
+/src/transformers/models/blip-2/*_modeling* @zucchini-nlp
+/src/transformers/models/bridgetower/*_modeling* @zucchini-nlp
+/src/transformers/models/bros/*_modeling* @zucchini-nlp
+/src/transformers/models/chameleon/*_modeling* @zucchini-nlp
+/src/transformers/models/chinese_clip/*_modeling* @zucchini-nlp
+/src/transformers/models/clip/*_modeling* @zucchini-nlp
+/src/transformers/models/clipseg/*_modeling* @zucchini-nlp
+/src/transformers/models/clvp/*_modeling* @zucchini-nlp
+/src/transformers/models/colpali/*_modeling* @zucchini-nlp
+/src/transformers/models/data2vec/*_modeling* @zucchini-nlp
+/src/transformers/models/deplot/*_modeling* @zucchini-nlp
+/src/transformers/models/donut/*_modeling* @zucchini-nlp
+/src/transformers/models/flava/*_modeling* @zucchini-nlp
+/src/transformers/models/git/*_modeling* @zucchini-nlp
+/src/transformers/models/grounding-dino/*_modeling* @zucchini-nlp
```

```suggestion
/src/transformers/models/grounding_dino/*_modeling* @zucchini-nlp
```

there are several model paths with "-" instead of "_"

---

Fixed!